tests/backdrop/RolesCommandsTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @file
+ * PHPUnit tests for Bee Roles commands.
+ */
+
+use PHPUnit\Framework\TestCase;
+/**
+ * Test Bee's commands for managing users.
+ */
+class RolesCommandsTest extends TestCase {
+
+  /**
+   * Make sure that the roles command works.
+   */
+  public function test_roles_command_works() {
+    $output = shell_exec('bee roles');
+    $this->assertRegExp('/| administrator | \'administer blocks\' +|/', (string) $output);
+  }
+
+  /**
+   * Make sure that the permissions command works.
+   */
+  public function test_permissions_command_works() {
+    $output = shell_exec('bee permissions');
+    $this->assertRegExp('/| block | \'administer blocks\' +|/', (string) $output);
```

Comment:
As per requested adjustment of `permissions_bee_callback()`

```suggestion
    $this->assertStringContainsString("BLOCK: 'administer blocks'", (string) $output);
```